service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -535,7 +532,7 @@ void createGetUpdateDeleteBqDataset() throws Exception {
             .build();
     ControlledBigQueryDatasetResource createdDataset =
         controlledResourceService.createBigQueryDataset(
```

Comment:
maybe refactor `createBigQueryDataset` to make the nullable parameter last (makes overloading easier).

Reply:
We can do this in another PR